substrate/bin/utils/chain-spec-builder/src/lib.rs

```diff
@@ -158,6 +161,12 @@ pub struct CreateCmd {
 	/// The chain type.
 	#[arg(value_enum, short = 't', default_value = "live")]
 	chain_type: ChainType,
+	/// The para ID for your chain.
+	#[arg(long, value_enum, short = 'p')]
```

Comment:
```suggestion
	#[arg(long, value_enum, short = 'p', requires = "relay_chain")]
```

If you add this, using `--relay-chain` cli arg require `--para-id` too, which is what we want. Using one of them without the other was silently swallowing.